pkg/request/batchstore.go

```diff
@@ -7,36 +7,34 @@ package request
 
 import (
 	"context"
-	"fmt"
 	"sync"
 	"sync/atomic"
-	"time"
 )
 
 type BatchStore struct {
-	batchingEnabled uint32
-	currentBatch    *batch
-	readyBatches    []*batch
-	onDelete        func(key string)
-	batchMaxSize    uint32
-	maxCapacity     int
-	keys2Batches    sync.Map
-	lock            sync.RWMutex
-	signal          sync.Cond
+	currentBatch      *batch
+	readyBatches      []*batch
+	onDelete          func(key string)
+	batchMaxSize      uint64
```

Comment:
Why did you decide to change it to 64 bits? aren't 32 bits enough?

Reply:
Again that is the way it is declared in the configuration file
Should I change it there as well?

Reply:
no need